example/html/js-payments-card/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
```

Comment:
Is there a way we can avoid duplicating these example pages? I.e. can the merchant docker use the example pages generated by the `npm run prod` instead of having to have a duplicate version of them in the testing library. 

I just think it'd be better not to have to update both if we want to add another component for example